cirq-web/cirq_ts/utils/ss_bloch_sphere.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 The Cirq Developers
```

Comment:
do we need this file and the utils folder? 
It seems like this was an experiment for the e2e tests.

Reply:
I had it as a utility for adding new screenshots for e2e tests as you updated the bloch sphere, but I also think that's something that users can do themselves.